vars/samDeploy.groovy

```diff
@@ -44,6 +49,11 @@ def call(body) {
   if (config.noFailOnEmptyChangeset != null && config.noFailOnEmptyChangeset == "true") {
     options = options.concat(" --no-fail-on-empty-changeset")
   }
+ 
+  def capabilities = config.get('capabilities', ['CAPABILITY_IAM'])
+  if (capabilities) {
```

Comment:
Is the if condition necessary when there is a default, therefore always defined? What would be the use case for not specifying capabilities?

Reply:
the use case here is to set the value of capabilities to false to disable the default capabilities